server.js

```diff
@@ -0,0 +1,10 @@
+var express = require('express')
```

Comment:
I've added this file to give them a placeholder for API routes if they want them.